test-deno.js

```diff
@@ -0,0 +1,3 @@
+import expect from './unexpected-deno.js';
+
```

Comment:
Ideally this would execute the full battery of tests we have - got some early work in the direction of trying to get the real suite running.